### PR TITLE
HWP-266 create test file for /login

### PR DESCRIPTION
**Pull request by dbarkowsky**
# Description

This PR includes the following proposed change(s):

Added test file for login functions.

## Requires

Add 'Requires Attention' label if appropriate.

- [ ] Requires an update to the .env file
- [ ] Requires adding a file listed in the .gitignore
- [ ] Requires an opinion or answer to a question, see comments.
- [ ] Requires other attention, see below.

If there are any requirements for the developer to make after this PR is merged please list them here.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

For pull requests that are not just a simple fix, please check that the branch works on your local machine.

- [ ] Tested by Zach.
- [ ] Tested by Brandon.
- [ ] Tested by Brady.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] If attention is required by the developer such as updating the .env file, these requirements have been listed.

### CODE REVIEW

File: app/server/tests/api-login.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        let refreshToken = response.body.refreshToken;
+
+        expect(token).toBeTruthy();
+        expect(typeof token).toBe('string');
+        expect(refreshToken).toBeTruthy();
+        expect(typeof refreshToken).toBe('string');
+    });
+});
+
+describe('Testing sub-optimal login input', () => {
```
Comment (akroon3r) [inline app/server/tests/api-login.test.js:29]: what is sub optimal?

Comment (dbarkowsky) [reply app/server/tests/api-login.test.js:29]: Was thinking inputs that the ideal user doesn't put in. Could change to something like correct vs incorrect inputs. Thoughts?

Comment (akroon3r) [reply app/server/tests/api-login.test.js:29]: what is the definition for correct vs incorrect?

Comment (dbarkowsky) [reply app/server/tests/api-login.test.js:29]: Whether or not the response is expected to be successful.

Comment (dbarkowsky) [reply app/server/tests/api-login.test.js]: @akroon3r  Updated descriptions. Is that clearer?

File: app/server/tests/api-login.test.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const { AuthFunctions } = require('./functions/authFunctions')
 
 let users = new AuthFunctions(); // build class for user actions
 
-describe('Testing optimal inputs for login', () => {
+describe('Testing inputs that are expected to succeed login', () => {
```
Comment (akroon3r) [inline app/server/tests/api-login.test.js:5]: Should we outline in a comment above what the constraints actually are?

Comment (dbarkowsky) [reply app/server/tests/api-login.test.js:6]: Added some comments.

**Review by akroon3r — APPROVED** [review]

**Pull request merged**